### DISCUSSION TITLE
selected text intermittently not colorized as configured

### DISCUSSION THREAD

**Discussion by tksoh** [eteran/nedit-ng#269] [General]
Just to keep a record of my observation.

I was working on the highlighting patterns on Windows for CMAKE. After closing the highlight pattern dialog, I went back to scroll the cmakes file to check for correctness. Then I notice the selection are showing the white color. 

I opened the config.ini and saw:

> nedit.colorizeHighlightedText=true

The problem went away after I restarted NG. I will keep monitoring this.

Using version 53a5d93169e6d0a8ea7bf6398240824a73430539

**Comment by eteran** [comment]
Hmm, OK, I'll double-check the code to see if anything stands out. But IIRC, it's just a simple `if` statement in the text rendering function of `TextArea`.
